classes/services/PKPStatsServiceTrait.php

```diff
@@ -89,7 +89,7 @@ public function getEmptyTimelineIntervals(string $startDate, string $endDate, st
             $interval = 'P1M';
         } elseif ($timelineInterval === PKPStatisticsHelper::STATISTICS_DIMENSION_DAY) {
             $dateFormat = 'Y-m-d';
-            $labelFormat = PKPString::convertStrftimeFormat(Application::get()->getRequest()->getContext()->getLocalizedDateFormatLong());
+            $labelFormat =Application::get()->getRequest()->getContext()->getLocalizedDateFormatLong();
```

Comment:
@defstat, there's a missing space here, but the code auto-formatting tools should've taken care of it. In `pkp-lib`, we have [Captain Hook](https://github.com/captainhookphp/captainhook) installed to set up a git pre-commit hook; can you double-check that it's set up and working on your machine (e.g. on a personal fork)?